File: Frontend/src/types/supportPolicy.ts
export interface SupportPolicy {
  id: number;
  title: string;
  policyType: string;
  content: string;
  effectiveFrom: string;
  createdAt: string;
}

export interface SupportPolicyPayload {
  title: string;
  policyType: string;
  content: string;
  effectiveFrom: string;
}

File: AI.CustomerSupport.API/AI.CustomerSupport/Models/SupportPolicy.cs
namespace AI.CustomerSupport.API.Models
{
    public class SupportPolicy
    {
        public int Id { get; set; }

        public string Title { get; set; } = string.Empty;

        public string PolicyType { get; set; } = string.Empty;

        public string Content { get; set; } = string.Empty;

        public DateTime EffectiveFrom { get; set; }

        public DateTime CreatedAt { get; set; }
    }
}
